lib/ui/index.js

```diff
@@ -44,16 +43,23 @@ class UI {
         this.verbose = this.options.verbose;
         this.allowPrompt = this.options.allowPrompt && Boolean(this.stdout.isTTY);
 
-        // Add custom prompt module that uses the
-        // specified streams
-        this.inquirer = inquirer.createPromptModule({
-            input: this.stdin,
-            output: this.stdout
-        });
-
         CLIRenderer.ui = this;
     }
 
+    get inquirer() {
+        if (!this._inquirer) {
+            // Add custom prompt module that uses the
+            // specified streams
+            const inquirer = require('inquirer');
```

Comment:
I am curios, why we not use lazyRequire for this case and move the dep on top? 🤔

Reply:
was actually thinking about doing that - will move it.